### ISSUE TITLE
Tab Prefix Keeps Changing

### ISSUE THREAD

**Issue by B00MERzY2K** [sgtcaze/NametagEdit#122]
### What version/build of NametagEdit are you using? Type /ver NametagEdit
**4.1.3**

### What version of Bukkit/Spigot/PaperSpigot are you using? Type /ver
**Spigot 1.8.8**

### What plugins are you using? Type /plugins
**icJukeBox, WorldEdit, SpawnTP, Skript, Essentials, PlugMan, AdvancedBan, SkQuery, NametagEdit, SimpleTicketManager, ProtocolLib, WorldGuard, Multiverse-Core, NoCheatPlus, KitAdder, AdminEssentials, iDisguise, Multiverse-Inventories, WildSkript, EssentialsChat, SSnowHub, PermissionsEx, mcMMO, Multiverse-Portals, ColoredTexts, Vault, MassiveCore, TitleManager, Auctions, HolographicDisplays, CommandSigns, ChatFormat, Citizens, Factions, skRayFall, Shopkeepers, CommandNPC, ArchonCrates, Blacksmith, SimpleAutoBroadcast, FirstJoinPlus.**

### Please explain your issue?
**Whenever I press TAB, I see the prefix that I set next to my name. But whenever I type something, it completely changes. It is normally set to a dark blue and has my name completely visible, but when I type something in chat or I type a command, it changes to a light blue and it cuts off my name. However, if I type "/nte reload", it changes back to the way it was, until I type something in again, then it messes up.** _http://imgur.com/a/UTsHh_

### How do you reproduce the issue?
**I just type something in chat, whether it be a message or a command.**

### Is there an error(s) in the console? Please use: https://pastebin.com
**No, there isn't. I opened my console and typed "Hello" into the in-game chat, then I checked the console but there were no errors. All I saw was my "Hello" message and that was it. However, the issue still occurred.**

**Comment by sgtcaze** [comment]
Do you happen to have an NPC of yourself in the world?

**Comment by B00MERzY2K** [comment]
Yeah. I've got around 1 or 2 of them.

**UPDATE:**
I just disabled the plugin "Citizens" and typed "Hello" in chat, but the problem still continued. I also disabled all my Chat plugins and typed "Hello" again, and it still continued.

**UPDATE:**
I just resolved my issue! Apparently the plugin "ColoredTexts" doesn't like to work well with others. Though it was very frustrating and difficult to track down the source of my issue, I finally managed to do so.

Case closed! :)
